### DISCUSSION TITLE
Maximum propagation distance depending on the source

### DISCUSSION THREAD

**Discussion by Yacout** [Universite-Gustave-Eiffel/NoiseModelling#458] [Q&A] [answered]
Hello,
I would like to know if there's a way to change the maximum propagation distance in the Noise Level From Source depending on the source. For example, I would like to have a big road with a long propagation distance and all the others with another propagation distance, all this in the same geojson file.
Thank you for your answer,

**Comment by nicolas-f** [comment]
Hi,
Why do you want to change the propagation distance for the big road ?
NoiseModelling is computing stuff from the perspective of receivers, so it will fetch sources by using the Maximum propagation distance parameter. It would be possible if it were the other way around (from source to receiver), but it is not the case.

**Comment by pierromond** [comment, marked as answer]
Hi @Yacout,

You can also divide your input dataset and run x simulations with x maximum distances. It could be a solution.

But as @nicolas-f said, this possibility is not included as it in NoiseModelling.
